StationeersLaunchPad/LaunchPadConfig.cs

```diff
@@ -680,5 +698,697 @@ public static void ExportModPackage()
         Logger.Global.LogException(ex);
       }
     }
+
+    public static bool IsActive = false;
+
+    public static void Draw()
+    {
+      ImGuiHelper.Draw(() => DrawConfigEditor());
+    }
+
+    public static bool ConfigChanged = false;
+    private static ModInfo draggingMod = null;
+    private static int draggingIndex = -1;
+    public static void DrawConfigTable(bool edit = false)
+    {
+      if (!ImGui.IsMouseDown(ImGuiMouseButton.Left))
+        draggingMod = null;
+
+      var hoveringIndex = -1;
+      if (draggingMod != null)
+        draggingIndex = Mods.IndexOf(draggingMod);
+
+      if (!edit)
+        ImGui.BeginDisabled();
+
+      if (ImGui.BeginTable("##configtable", 3, ImGuiTableFlags.SizingFixedFit))
+      {
+        ImGui.TableSetupColumn("##enabled");
+        ImGui.TableSetupColumn("##type");
+        ImGui.TableSetupColumn("##name", ImGuiTableColumnFlags.WidthStretch);
+
+        for (var i = 0; i < Mods.Count; i++)
+        {
+          var mod = Mods[i];
+
+          ImGui.TableNextRow();
+          ImGui.PushID(i);
+          ImGui.TableNextColumn();
+
+          if (ImGui.Checkbox("Enabled", ref mod.Enabled))
```

Comment:
the Enabled text here takes up a good amount of space and the function of the checkbox is fairly straightforward without it. could we switch back to just the checkbox here?